metrics/mixin/alerts/mirroring.libsonnet

```diff
@@ -23,7 +23,7 @@
           {
             alert: 'OdfPoolMirroringImageHealth',
             expr: |||
-              (ocs_pool_mirroring_image_health{%(ocsExporterSelector)s}  * on (namespace) group_left() (max by(namespace) (ocs_pool_mirroring_status{%(ocsExporterSelector)s}))) == 1
```

Comment:
The problem is, even after disabling the mirroring, the old mirroring status is displayed on Blackpool CR. So ```ocs_pool_mirroring_image_health``` will keep reporting the old health status. Here, a query expression is written like if one pool is enabled for mirroring then all other pools which have old mirroring status also get triggering an alert.

CR has old status or new status is does not matter, but a mirroring alert has to be triggered only when mirroring is enabled for the corresponding pool. so I am checking namespace as well as name while grouping.

Reply:
@vasukulkarni